kubernetes/yamls/development/elasticsearch/es-rc.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: v1
+kind: ReplicationController
```

Comment:
I took the reference from [Kubernetes example for ES](https://github.com/kubernetes/examples/tree/master/staging/elasticsearch) and this comes from there. Will make the changes.